draft-tjhai-ipsecme-hybrid-qske-ikev2-02.xml

```diff
@@ -383,10 +415,51 @@
 		needs IKE to handle any possible fragmentation, then he uses the protocol
 		listed below.</t>
 
+    <section title="IKE_SA_INIT Round: Negotiation" >
+        <t> Using multiple key exchanges is negotiated by means of standard IKEv2 mechanism - via SA payload.
+        For this purpose several new transform types, namely Additional Key Exchange 1, Additional Key Exchange 2,
+        Additional Key Exchange 3 etc. are defined. They are collectively called Additional Key Exchanges 
+        and have slightly different semantics than existing IKEv2 transform types.
+        They are interpreted as additional key exchanges that peers agreed to perform
+        in a series of INTERMEDIATE exchanges. The possible transform IDs for these transform types 
+        are the same as IDs for the transform type 4 (Diffie-Hellman Group), so they all share
```

Comment:
Shall we be consistent with `Transform Type N` or `transform type N` when we are specifying a particular transform type? I personally prefer `Transform Type N`.

Reply:
Agree.